packages/unenv-preset/src/preset.ts

```diff
@@ -64,29 +65,25 @@ export const cloudflare: Preset = {
 		),
 
 		// The `node:sys` module is just a deprecated alias for `node:util` which we implemented using a hybrid polyfill
-		sys: "@cloudflare/unenv-preset/runtime/node/util/index",
-		"node:sys": "@cloudflare/unenv-preset/runtime/node/util/index",
+		sys: "@cloudflare/unenv-preset/node/util",
+		"node:sys": "@cloudflare/unenv-preset/node/util",
 
 		// define aliases for hybrid modules
 		...Object.fromEntries(
 			hybridNodeCompatModules.flatMap((m) => [
-				[m, `@cloudflare/unenv-preset/runtime/node/${m}/index`],
-				[`node:${m}`, `@cloudflare/unenv-preset/runtime/node/${m}/index`],
+				[m, `@cloudflare/unenv-preset/node/${m}`],
+				[`node:${m}`, `@cloudflare/unenv-preset/node/${m}`],
 			])
 		),
-
-		// TODO: this is a hotfix and breaks unenv/fetch
-		// https://github.com/unjs/unenv/issues/364
-		"unenv/runtime/node/stream/index": "node:stream",
 	},
 	inject: {
-		// workerd already defines `global` and `Buffer`
-		// override the previous presets so that we use the native implementation
```

Comment:
I quite liked this comment.

But perhaps you removed it because we are no longer piggy backing on top of the nodeless preset?
In which case can we get rid of

```
		Buffer: false,
		global: false,
```

too?

Reply:
I added back `// Setting symbols implemented by workerd to `false` so that `inject`s defined in base presets are not used.`